test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -280,7 +286,14 @@
           [foo-service baz-service fail-service]
           (let [resp (http-client/get "http://localhost:8180/status/v1/simple")]
             (is (= 503 (:status resp)))
-            (is (= "error" (slurp (:body resp)))))))
+            (is (= "unknown" (slurp (:body resp)))))))
```

Comment:
@nathanielksmith note this test change: previously if any service reported an error, /simple would return :error. Based on our discussion in hipchat I've changed it to return :unknown in that case. The behavior now agrees with the summarize-states docstring, but we should be sure it's what we want.